src/main/java/com/courtcanva/ccfranchise/services/StaffService.java

```diff
@@ -22,4 +23,12 @@ public StaffGetDto createStaff(Staff staff) {
 
     }
 
+    public Boolean emailExists(String email) {
+        Boolean isExisted = staffRepository.existsStaffByEmail(email);
+        if (Boolean.TRUE.equals(isExisted)) {
```

Comment:
is `if (existed) { ` enough?

Reply:
It is enough, cause I have added @Email as validation to prevent invalid email input.